rental.rb

```diff
@@ -0,0 +1,10 @@
+# Represents the rental class
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
```

Comment:
- Hey, great effort here, now we also need to add the instances on pushing to the the person and the book. update it by adding the following below:

```ruby
    person.rentals << self
    book.rentals << self
```